pages/common/salt-key.md

```diff
@@ -2,6 +2,7 @@
 
 > Manages salt minion keys on the salt master.
 > Needs to be run on the salt master, likely as root or with sudo.
+> Homepage: <https://www.saltstack.com>.
```

Comment:
```suggestion
> Homepage: <https://docs.saltstack.com/en/latest/ref/cli/salt-key.html>.
```

Reply:
I'll update this for `salt-call`, `salt-key` and `salt-run`.

Just wondering, should it use https://docs.saltstack.com/ref/cli/salt-key.html or include the `/en/latest` in the URL (which is automatically redirected).

Reply:
If they're equivalent, I'd go with the shorter URL.

Reply:
Maybe this should be `Documentation:`, since it links to the CLI reference?

Reply:
@sbrl replied [below](https://github.com/tldr-pages/tldr/pull/3033#issuecomment-493329994).